.github/workflows/node.js.yml

```diff
@@ -20,6 +20,17 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Setup unit test environment
+      run: |-
+        sudo apt-get update -y
+        sudo apt-get install -y libwebp-dev libcairo2-dev libopenjp2-tools exiftool
```

Comment:
we should instead use a docker image with these already installed (if possible) so the versions are stable

Reply:
was trying to come up with something like that but figure out it will take much more time cuz when we cannot use image from internal registry here. And there's not an existing place for us to push a pre-build image as such just for testing.

Reply:
we actually have one that we use in the sdk: https://github.com/adobe/asset-compute-sdk/blob/master/test-worker/manifest.yml#L7
it has imagemagick and exiftool I believe

Reply:
+1. 
Allthough the node version is older (node10). There is a story to update that image to use node14...